### DISCUSSION TITLE
Linkerd connect: connection refused

### DISCUSSION THREAD

**Discussion by sanyam42** [linkerd/linkerd2#8654] [Help]
Liveness probe and Readiness probe are getting failed for linkerd proxy container in our application. Linkerd control planes pods are running fine.

Liveness probe failed: Get http://100.64.76.153:4191/live: dial tcp 100.64.76.153:4191: connect: connection refused

Linkerd Server version: stable-2.11.1

**Comment by adleong** [comment]
I recommend running through the troubleshooting steps documented here: https://linkerd.io/2.11/tasks/troubleshooting/
